resources/build/trigger-builds.sh

```diff
@@ -25,14 +25,26 @@ function triggerBuild() {
   local TEAMCITY_BUILDTYPE="$1"
   local TEAMCITY_VCS_ID="$2"
 
+  if [[ $# -gt 2 ]]; then
+    local TEAMCITY_BRANCH_NAME="$3"
+    #debug echo "  Triggering build for: $TEAMCITY_BRANCH_NAME"
```

Comment:
Wait... so we don't want to have the triggered builds explicit in the Trigger Tests CI log?  Or is it already explicit through the console commands that will show up in the CI log?

While I'm fine leaving the `echo "Call: $command"` line as a debug comment, I think leaving this `echo` "in production" is wise - it's cleaner than the other while still allowing for easier debugging from log notes if needed.

Reply:
The log does already show the triggered builds (from the caller) plus the result of the curl call:
```
[09:30:20][Step 2/2]   -- Triggering build configuration KeymanAndroid_TestPullRequests
[09:30:20][Step 2/2] {"id":181705,"buildTypeId":"KeymanAndroid_TestPullRequests","state":"queued","branchName":"2611",...<snip>...
[09:30:20][Step 2/2]  -- Triggering build configuration KeymanAndroid_TestSamplesAndTestProjects
[09:30:20][Step 2/2] {"id":181706,"buildTypeId":"KeymanAndroid_TestSamplesAndTestProjects","state":"queued","branchName":"2611",...<snip>...
[09:30:20][Step 2/2] # ios: changes detected
```